wuzz.go

```diff
@@ -1016,7 +1055,22 @@ func main() {
 
 	initApp(app, g)
 
-	err = app.ParseArgs(g)
+	// load config (must be done *before* app.ParseArgs, as arguments
+	// should be able to override config values). An empty string passed
+	// to LoadConfig results in LoadConfig loading the default config
+	// location. If there is no config, the values in
+	// config.DefaultConfig will be used.
+	err = app.LoadConfig(configPath)
+	if err != nil {
+		log.Fatalf("Error loading config file: %v", err)
```

Comment:
`gocui.Gui` is initialized here, so it need to be explicitly closed by calling `g.Close()` before printing - see line 1075.